integration_tests/tests/apply/apply.cy.ts

```diff
@@ -147,6 +148,12 @@ context('Apply', () => {
     placementStartPage.checkRadioByNameAndValue('startDateSameAsReleaseDate', 'yes')
     placementStartPage.clickSubmit()
 
+    const placementPurposePage = new PlacementPurposePage()
+    placementPurposePage.checkCheckboxByNameAndValue('placementPurpose', 'drugAlcoholMonitoring')
+    placementPurposePage.checkCheckboxByNameAndValue('placementPurpose', 'otherReason')
+    placementPurposePage.getTextInputByIdAndEnterDetails('otherReason', 'Another reason')
```

Comment:
My instinct is that the specific checkboxes would live inside PlacementPurposePage, so `apply.cy` doesn't need to know about the checkbox names, but if this is the standard pattern, no objection